### ISSUE TITLE
[llvm] build failure

### ISSUE THREAD

**Issue by learn-more** [microsoft/vcpkg#11916]
It seems that not installing the package-default clang leads to problems in the (?install?) step.

**Host Environment**
- OS: Windows 10, 1903
- Compiler: MSVC 19.25.28614.0

**To Reproduce**
`./vcpkg.exe install --triplet x64-windows-static llvm[core,disable-assertions,target-arm,target-aarch64,target-mips,target-powerpc,target-sparc,target-x86]`

**Failure logs** 
```
Starting package 7/9: llvm:x64-windows-static
Building package llvm[core,disable-assertions,target-aarch64,target-arm,target-mips,target-powerpc,target-sparc,target-x86]:x64-windows-static...
-- Downloading https://github.com/llvm/llvm-project/archive/llvmorg-10.0.0.tar.gz...
-- Extracting source E:/Projects/rehex/vcpkg/vcpkg/downloads/llvm-llvm-project-llvmorg-10.0.0.tar.gz
-- Applying patch 0001-allow-to-use-commas.patch
-- Applying patch 0002-fix-install-paths.patch
-- Applying patch 0003-fix-vs2019-v16.6.patch
-- Using source at E:/Projects/rehex/vcpkg/vcpkg/buildtrees/llvm/src/org-10.0.0-32c002c70c
-- Downloading https://www.python.org/ftp/python/3.8.3/python-3.8.3-embed-amd64.zip...
-- Configuring x64-windows-static
-- Building x64-windows-static-dbg
-- Building x64-windows-static-rel
CMake Error at scripts/cmake/vcpkg_fixup_cmake_targets.cmake:116 (message):

  'E:/Projects/rehex/vcpkg/vcpkg/packages/llvm_x64-windows-static/debug/share/clang'
  does not exist.
Call Stack (most recent call first):
  ports/llvm/portfile.cmake:201 (vcpkg_fixup_cmake_targets)
  scripts/ports.cmake:76 (include)


Error: Building package llvm:x64-windows-static failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: llvm:x64-windows-static
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.
```

Some relevant files from `E:\Projects\rehex\vcpkg\vcpkg\buildtrees\llvm`:
[stdout-x64-windows-static.log](https://github.com/microsoft/vcpkg/files/4774575/stdout-x64-windows-static.log)
[x64-windows-static.vcpkg_abi_info.txt](https://github.com/microsoft/vcpkg/files/4774577/x64-windows-static.vcpkg_abi_info.txt)

**Comment by LilyWangL** [comment]
Thanks for posting this issue. I can't reproduce this issue locally, did you do some changes before building llvm?

**Comment by learn-more** [comment]
Looking at the activity on this ticket, I can ignore the email about not being able to reproduce?

**Comment by LilyWangL** [comment]
Yes, this issue will be fixed in PR #11956.
